controllers/storagecluster/cephcluster.go

```diff
@@ -405,6 +410,9 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, s
 		MaxLogSize:  &maxLogSize,
 	}
 
+	osdStore := getOSDStoreConfig(sc)
+	reqLogger.Info("osd store settings", osdStore)
```

Comment:
For the non-rdr case, will this setting be blank? Perhaps only log this if not blank?

Reply:
yes. Logging only if not blank.